views/skus/edit.erb

```diff
@@ -0,0 +1,81 @@
+<div class="container">
+  <legend> Edit SKU </legend>
+  <form action='/skus' method='post'>
+    <input type="hidden" name="_method" value="patch">
+    <input type="hidden" name="id" value="<%= @sku.id %>">
+    <div class="form-group row">
+      <label class="col-md-offset-3 col-md-5 control-label col-md-offset-3">Name</label>
+      <div class="col-md-offset-3  col-md-5 inputGroupContainer col-md-offset-3">
+        <div class="input-group">
+          <span class="input-group-addon">
+            <i class="glyphicon glyphicon-book"></i>
+          </span>
+          <input name="name" placeholder="<%= @sku.name %>" class="form-control" type="text">
+        </div>
+      </div>
+    </div>
+
+    <div class="form-group row">
+      <label class="col-md-offset-3 col-md-5 control-label col-md-offset-3">Number</label>
+      <div class="col-md-offset-3  col-md-5 inputGroupContainer col-md-offset-3">
+        <div class="input-group">
+          <span class="input-group-addon">
+            <i class="glyphicon glyphicon-tree-conifer"></i>
+          </span>
+          <input name="sku_num" placeholder="<%= @sku.sku_num %>" class="form-control"  type="text">
+        </div>
+      </div>
+    </div>
+
+    <div class="form-group row">
+      <label class="col-md-offset-3 col-md-5 control-label col-md-offset-3">Family</label>
+      <div class="col-md-offset-3  col-md-5 inputGroupContainer col-md-offset-3">
+        <div class="input-group">
+          <span class="input-group-addon">
+            <i class="glyphicon glyphicon-tree-conifer"></i>
+          </span>
+          <input name="family" placeholder="<%= @sku.family %>" class="form-control"  type="text">
+        </div>
+      </div>
+    </div>
+
+    <div class="form-group row">
+      <label class="col-md-offset-3 col-md-5 control-label col-md-offset-4">Description</label>
+      <div class="col-md-offset-3  col-md-5 inputGroupContainer col-md-offset-4">
+        <div class="input-group">
+          <span class="input-group-addon">
+            <i class="glyphicon glyphicon-tree-conifer"></i>
+          </span>
+          <input name="description" placeholder="<%= @sku.description %>" class="form-control"  type="text">
```

Comment:
Hey @hsolorza , I made some changes to the edit.erb, I reverted it back so that it matches the other edit pages on the website, to keep things consistent. I went through all the requested chagnes here and made them as well!